app/views/templates/package_action.erb

```diff
@@ -0,0 +1,46 @@
+<%#
+kind: job_template
+name: Package Action - SSH Default
+job_name: PackageAction
```

Comment:
Any reason fro not usnig space in the name? 'Package Action' would look more nicely in the UI